constant does not match type

https://github.com/epics-base/pvAccessCPP/blob/1492b3d5af0a17802a8c904947a62578190af54e/src/pva/pv/pvaConstants.h#L76

`0xFFFF` does not fit in an `int16`. MSVC complains with the warning
```
src\pva\pv\pvaConstants.h(76): warning C4309: 'initializing': truncation of constant value
```

Consider either using the data type `uint16` or the value `-1` or remove the constant altogether as it seems not to be used anywhere.

--- a/src/pva/pv/pvaConstants.h
+++ b/src/pva/pv/pvaConstants.h
@@ -73,7 +73,7 @@ const epics::pvData::int16 PVA_DEFAULT_PRIORITY = 0;
 const epics::pvData::uint32 MAX_CHANNEL_NAME_LENGTH = 500;
 
 /** Invalid data type. */
-const epics::pvData::int16 INVALID_DATA_TYPE = 0xFFFF;
+const epics::pvData::int16 INVALID_DATA_TYPE = -1;
 
 /** Invalid IOID. */
 const epics::pvData::int32 INVALID_IOID = 0;

--- a/src/utils/inetAddressUtil.cpp
+++ b/src/utils/inetAddressUtil.cpp
@@ -38,7 +38,7 @@ void encodeAsIPv6Address(ByteBuffer* buffer, const osiSockAddr* address) {
     buffer->putLong(0);
     buffer->putShort(0);
     // next 16-bits are 1
-    buffer->putShort(0xFFFF);
+    buffer->putShort(-1);
     // following IPv4 address in big-endian (network) byte order
     uint32_t ipv4Addr = ntohl(address->ia.sin_addr.s_addr);
     buffer->putByte((int8)((ipv4Addr>>24)&0xFF));

